travel.html

```diff
@@ -106,7 +106,7 @@ <h3 id="travel-subsidies">Travel Subsidies</h3>
 
             <h3 id="arriving-by-car">Arriving by Car</h3>
 
-            <p>Please park in the North Garage (<a href="https://goo.gl/maps/grzxysUX2jG2">map</a>), located near Friend Center at the northeast corner of campus. If the garage is full, you may also park in the Ivy Lane parking lots or Lot 21. Parking in any of these lots is free and you will not need to display a parking permit or overnight permit. For a detailed parking map, go to <a href="http://www.princeton.edu/parking/Parking_Lots.pdf">http://www.princeton.edu/parking/Parking_Lots.pdf</a>.</p>
+            <p>Please park in the North Garage (<a href="https://goo.gl/maps/grzxysUX2jG2">map</a>), located near Friend Center at the northeast corner of campus. If the garage is full, you may also park in the Ivy Lane parking lots or Lot 2. Parking in any of these lots is free and you will not need to display a parking permit or overnight permit. Lots 21, 4, 5, 25, and Western Way will not be open. For a detailed parking map, go to <a href="http://www.princeton.edu/parking/Parking_Lots.pdf">http://www.princeton.edu/parking/Parking_Lots.pdf</a>.</p>
```

Comment:
should make it more obvious that this is *not* the same as previous years since Lot 21 was almost always the lot we used